collectors/COLLECTORS.md

```diff
@@ -1,219 +1,478 @@
+-   [Postgres](/collectors/python.d.plugin/postgres/README.md): Collect database health and performance metrics. 
+-   [ProxySQL](/collectors/python.d.plugin/proxysql/README.md): Monitor database backend and frontend performance
+    metrics.
+-   [Redis](/collectors/python.d.plugin/redis/): Monitor database status by reading the server's response to the `INFO`
+    command.
+-   [RethinkDB](/collectors/python.d.plugin/rethinkdbs/README.md): Collect database server and cluster statistics.
+-   [Riak KV](/collectors/python.d.plugin/riakkv/README.md): Collect database stats from the `/stats` endpoint.
+-   [Zookeeper](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/zookeeper/): Monitor application
+    health metrics reading the server's response to the `mntr` command.
+
+### Distributed computing
+
+-   [BOINC](/collectors/python.d.plugin/boinc/README.md): Monitor the total number of tasks, open tasks, and task
+    states for the distributed computing client.
+-   [Gearman](/collectors/python.d.plugin/gearman/README.md): Collect application summary (queued, running) and per-job
+    worker statistics (queued, idle, running).
+
+### Email
+
+-   [Dovecot](/collectors/python.d.plugin/dovecot/README.md): Collect email server performance metrics by reading the
+    server's response to the `EXPORT global` command.
+-   [EXIM](/collectors/python.d.plugin/exim/README.md): Uses the `exim` tool to monitor the queue length of a
+    mail/message transfer agent (MTA).
+-   [Postfix](/collectors/python.d.plugin/postfix/README.md): Uses the `postqueue` tool to monitor the queue length of a
+    mail/message transfer agent (MTA).
+
+### Kubernetes
+
+-   [Kubelet](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubelet/): Monitor one or more
+    instances of the Kubelet agent and collects metrics on number of pods/containers running, volume of Docker
+    operations, and more.
+-   [kube-proxy](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubeproxy/): Collect
+    metrics, such as syncing proxy rules and REST client requests, from one or more instances of `kube-proxy`.
+-   [Service discovery](https://github.com/netdata/agent-service-discovery/): Find what services are running on a
+    cluster's pods, converts that into configuration files, and exports them so they can be monitored by Netdata.
+
+### Logs
+
+-   [Fluentd](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/fluentd/): Gather application
+    plugins metrics from an endpoint provided by `in_monitor plugin`.
+-   [Logstash](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/logstash/): Monitor JVM threads,
+    memory usage, garbage collection statistics, and more.
+-   [OpenVPN status logs](/collectors/python.d.plugin/ovpn_status_log/): Parse server log files and provide summary
+    (client, traffic) metrics.
+-   [Web server logs (Go version for Apache,
```

Comment:
we have separate go collector for squid logs

> https://github.com/netdata/go.d.plugin/tree/master/modules/squidlog

Reply:
Added.